protocol/avdl/stellar1/local.avdl

```diff
@@ -134,6 +136,8 @@ protocol local {
     string sourceAmountMax;
     string sourceAmountActual;
     Asset sourceAsset;
+    double sourceToDestinationConversionRate; // The amount of the destination asset worth 1 unit of the source asset at the time of the transaction
```

Comment:
Good point. I think conversion rates always require a bit of thinking. So the name doesn't have to encode the direction, and the comment is helpful. `sourceConverstionRate` sounds fine. Other options: `pathConvRate`, `pathConv`, `convRate`.

Storing amounts in strings is lossless. No matter what language you run it through, nothing about the amount will be changed. Storing amounts in floating point is susceptible to a number of problems and should be avoided. For example, some arithmetic will yield inaccurate results. Storing amounts in an integer is better, but still has pitfalls. Different languages and transport protocols may drop bits off an int64.

So keep playing it safe and use strings for representing amounts. And if any manipulations need to be done, do them in go. We have existing functions with tests like `func FormatAmount`.

Btw if the value is actually being changed as in rounding then I like to use `amount` (or `rate`) to mean the precise amount. And `amountDescription` or `amountDesc` (`rateDesc`) to indicate a human-readable value that may be slightly inaccurate. To avoid a potential bug where non-gui code would ingest a rounded value.